unity-environment/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WalkerAgent : Agent
+{
+    [Header("Specific to Walker")] 
+    public Vector3 goalDirection;
+    public Transform hips;
+    public Transform chest;
+    public Transform spine;
+    public Transform head;
+    public Transform thighL;
+    public Transform shinL;
+    public Transform footL;
+    public Transform thighR;
+    public Transform shinR;
+    public Transform footR;
+    public Transform armL;
+    public Transform forearmL;
+    public Transform handL;
+    public Transform armR;
+    public Transform forearmR;
+    public Transform handR;
+    public Dictionary<Transform, BodyPart> bodyParts = new Dictionary<Transform, BodyPart>();
```

Comment:
Variable name `bodyParts` implies that this is a list of bodyPart objects, not a dictionary.